index.html

```diff
@@ -55,7 +55,16 @@ <h3>Sreelakshmi Jayarajan</h3>
       <a href="https://twitter.com/swsrkty">Twitter</a> |
       <a href="https://github.com/swsrkty">GitHub</a>
       </p>
-      <h3>Vishnu Sanal</h3>
+      <h3>Vishnu Sanal. T</h3>
+      <p>
+        Member at Moonshot, and a B.Tech CSE (S4) student at GEC Palakkad. These days he's working on <a href="https://www.GitHub.com/VishnuSanal/">open-source projects</a> including <a href="https://www.GitHub.com/TeamAmaze/AmazeFileManager">AmazeFileManager</a>, <a href="https://www.GitHub.com/VishnuSanal/DialogMusicPlayer">DialogMusicPlayer</a> & <a href="https://www.GitHub.com/VishnuSanal/QuotesStatusCreator">QuotesStatusCreator</a>. Previously, he was an intern at <a href="https://github.com/RobotutorLLC">RobotutorLLC</a> and have volunteered as a mentor for FOSS OpenHack '20.
```

Comment:
- You sure you want to link to Robotutor's GitHub not their landing page?

Reply:
> You sure you want to link to Amaze's GitHub not their landing page / play store page?

I think GitHub would be good since it is an OSS project. What do you think?

> Anyway, Amaze is quite popular, so not sure if it even needs a link (for example, I doubt you would put a link to Twitter if you were working at Twitter but I understand this is getting more philosophical, haha).

A link would be good, I think. And Amaze ain't popular as Twitter :D

> I think the above applies for Quotes, Dialog and Robotutor, but your wish

Oh, but I changed the link to RT's landing page as you suggested. But I think I should link to QSC & DMP. What do you think? How about removing all links to my GitHub since it is already present below?

> Missed a space between the words in AmazeFileManager (I think it's Amaze File Manager)

Yep.

> Missed a space between the words in DialogMusicPlayer

Nops.

Reply:
Sounds good